src/nih_wayfinding/app/scripts/navigation/navigation-service.js

```diff
@@ -0,0 +1,111 @@
+
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function Navigation (
+        $q, $interval, $rootScope,
+        $geolocation,
+        NavigationStub
+    ) {
+
+        var stepsLeft = [];
+        var navInterval = 1000; // 1 second default interval
```

Comment:
Missed this earlier, but for clarity consider including the unit in the variable name, in this case 'navIntervalMillis'